resources/data/ses_config-0.1.2.json

```diff
@@ -57,6 +57,20 @@
                     ]
                 }
             ]
+        },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "title": "Read user guides",
+                    "description": [
+                        "See the following user guides in the nRF Connect SDK documentation for essential information about developing applications for your device:",
+                        "- [Working with nRF9160 samples](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160.html)",
```

Comment:
We decided to rename this to "Working with nRF9160" to be consistent.

Reply:
Yes, that looks better. I adopted that renaming in d9b145b.

Do you already have a schedule when the user guide https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf5340.html will be available?

Reply:
The plan is Thursday morning.